harmonica/forward/point_mass.py

```diff
@@ -172,22 +187,16 @@ def point_mass_gravity(
     cast = np.broadcast(*coordinates[:3])
     result = np.zeros(cast.size, dtype=dtype)
     # Prepare arrays to be passed to the jitted functions
-    coordinates = tuple(np.atleast_1d(i).ravel() for i in coordinates[:3])
-    points = tuple(np.atleast_1d(i).ravel() for i in points[:3])
-    masses = np.atleast_1d(masses).ravel()
-    # Sanity checks
```

Comment:
This change makes the test `test_invalid_masses_array` useless. I will remove this test.

Reply:
Hi @birocoles, sorry for my disappearance, I was a little bit busy on the last weeks.

Why would you remove the sanity check for the point masses?